packages/lab/stories/static-list/static-list.qa.stories.tsx

```diff
@@ -49,17 +51,27 @@ export const AllExamples: StoryFn<QAContainerProps> = () => (
       {complexEventsData.map(({ title, time }) => (
         <StaticListItem key={title}>
           <StaticListItemContent>
-            <StackLayout gap={0.5}>
+            <StackLayout gap={0.5} id={`label-${id}`}>
               <Text color="inherit">{title}</Text>
               <Text styleAs="label" color="secondary">
                 {time}
               </Text>
             </StackLayout>
           </StaticListItemContent>
-          <Button appearance="transparent" aria-label="open in another tab">
+          <Button
+            id={`information-button-${id}`}
```

Comment:
It's quite small but my OCD... can aria-label and aria-labelledby be applied in a consistent order pls 😅